README.md

```diff
@@ -37,22 +37,28 @@ In order to use the device in user space, you need to import the provided [udev
 
 Run `yarp-omega3-server` and send RPC commands to `/yarp-omega3-server/rpc:i`.
 
+> please run the `yarpserver` if you did not already before running the `yarp-omega3-server`
+
 Available commands are
-- help
-- stop (completely disengange robot control)
-- quit (close the module)
-- set_position x y z (send a static position)
-- track_position x y z (send a position, to be used in _streaming_ mode)
-- set_force f_x f_y f_z (send a force reference)
+- `help`
+- `stop` (completely disengange robot control)
+- `quit` (close the module)
+- `set_position(x, y, z)` (send a static position)
+- `set_position_move_param(amax, vmax, jerk)` (limit movement parameters)
```

Comment:
Can you please add a description of these 3 parameters?

I guess `amax` is the maximum velocity and `vmax` is the maximum velocity. What is the `jerk` parameter instead? Why it is not `jerkmax` as the others two?

Also, if possible, I would change "limit movement parameters" to "set motion parameters".

Finally, I will specify that after sending this command the internal state of the server will **not** go back to the state before the call. Just to made the users aware.

Thank you